.github/workflows/python_ci.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Python CI
+on: [push, pull_request]
+
+jobs:
+  python-build:
+    name: Python Build
+    runs-on: ubuntu-latest
+    container: fedora:latest
```

Comment:
@lukash Any chance we could add a test matrix to test on CentOS 7 and CentOS 8 also?

Reply:
We could (assuming this still works on CentOS 7 which I'm not sure of?). If we do that it makes me wonder whether we should add more CI across the OS matrix and that in turn makes we wonder whether there can be too much CI :grin: